module.txt

```diff
@@ -1,7 +1,7 @@
 {
     "id" : "SimpleFarming",
     "version" : "1.0.0-SNAPSHOT",
-    "author" : "Josharias, patrick Wang",
+    "author" : "Josharias, patrick Wang, Harry Wang",
```

Comment:
Got a strange encoding issue on the marked whitespace at the beginning of the line when testing.
Eclipse also complains about invalid json syntax and the beginning of the line renders to ` Â   `.
Please double check.